<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibraryPage</class>
 <widget class="QWidget" name="LibraryPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>204</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="4">
    <widget class="LineEdit" name="search">
     <property name="toolTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="TreeView" name="view"/>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="libraryUpdate"/>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="genreCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="addToPlaylist"/>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="replacePlaylist"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>TreeView</class>
   <extends>QTreeView</extends>
   <header>treeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
